### PR TITLE
docs: add changelog for 4.1.0

### PR DESCRIPTION
**Pull request by vchimev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+* crash at application launch on Android P ([#5831](https://github.com/NativeScript/NativeScript/issues/5831)) ([7851629](https://github.com/NativeScript/NativeScript/commit/7851629))
+
+### Features
+
+* **builder:** enable reading xml from bundle ([#5668](https://github.com/NativeScript/NativeScript/issues/5668)) ([6fb7481](https://github.com/NativeScript/NativeScript/commit/6fb7481))
+* **file-system:** add ability to retrieve file size ([#5710](https://github.com/NativeScript/NativeScript/issues/5710)) ([6bdb5b5](https://github.com/NativeScript/NativeScript/commit/6bdb5b5))
+* **tabview:** add tab text font size property ([#5752](https://github.com/NativeScript/NativeScript/issues/5752)) ([11f0d6e](https://github.com/NativeScript/NativeScript/commit/11f0d6e))
+* **view:** introduce LayoutChanged event on every View component ([#5825](https://github.com/NativeScript/NativeScript/issues/5825)) ([0fc1547](https://github.com/NativeScript/NativeScript/commit/0fc1547))
+* CSS gradients ([#5534](https://github.com/NativeScript/NativeScript/issues/5534)) ([5a83a1c](https://github.com/NativeScript/NativeScript/commit/5a83a1c))
```
Comment (sis0k0) [inline CHANGELOG.md]: Maybe we can make this one a bit more descriptive:

**css**: add support for linear-gradient for the background and background-gradient properties

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+* **modal:** exception when calling ViewBase.showModal(...) ([#5737](https://github.com/NativeScript/NativeScript/issues/5737)) ([451589d](https://github.com/NativeScript/NativeScript/commit/451589d))
+* **modal:** innerView.closeModal(...) not passing back context ([#5833](https://github.com/NativeScript/NativeScript/issues/5833)) ([1365f13](https://github.com/NativeScript/NativeScript/commit/1365f13))
+* **navigation:** fix frame.navigate call inside page.navigatedTo handler ([#5649](https://github.com/NativeScript/NativeScript/issues/5649)) ([cf950e1](https://github.com/NativeScript/NativeScript/commit/cf950e1))
+* crash at application launch on Android P ([#5831](https://github.com/NativeScript/NativeScript/issues/5831)) ([7851629](https://github.com/NativeScript/NativeScript/commit/7851629))
+
+### Features
+
+* **builder:** enable reading xml from bundle ([#5668](https://github.com/NativeScript/NativeScript/issues/5668)) ([6fb7481](https://github.com/NativeScript/NativeScript/commit/6fb7481))
```
Comment (sis0k0) [inline CHANGELOG.md]: This was released with 4.0.1.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+* **flexbox-ios:** resolve text size issue for ListView with flexbox layout template ([#5799](https://github.com/NativeScript/NativeScript/issues/5799)) ([a03065c](https://github.com/NativeScript/NativeScript/commit/a03065c))
+* **flexbox:** label text wrapping inside flexbox layout ([#5781](https://github.com/NativeScript/NativeScript/issues/5781)) ([481043f](https://github.com/NativeScript/NativeScript/commit/481043f))
+* **ios-image-asset:** getImageAsync() can retrieve assets stored remotely in iCloud ([#5705](https://github.com/NativeScript/NativeScript/issues/5705)) ([9e3da8e](https://github.com/NativeScript/NativeScript/commit/9e3da8e))
+* **ios-searchbar:** fix searchbar auto sizing in iOS11 ([#5658](https://github.com/NativeScript/NativeScript/issues/5658)) ([555e592](https://github.com/NativeScript/NativeScript/commit/555e592))
+* **layout:** IOS Layout not invalidated with custom root ([#5724](https://github.com/NativeScript/NativeScript/issues/5724)) ([49ea10b](https://github.com/NativeScript/NativeScript/commit/49ea10b))
```
Comment (sis0k0) [inline CHANGELOG.md]: This was released with 4.0.1.

File: CHANGELOG.md (hunk 1 above)
Comment (ADjenkov) [reply CHANGELOG.md]: `background` and `backgroundImage`* properties

Comment (vchimev) [reply CHANGELOG.md]: Thank you! Updated. 😄

**Pull request merged**